src/resources/i18n/fr.json

```
@@ -142,7 +142,7 @@
     "referralfrom": "Visite prescrite par une autre unité hospitalière",
     "referralto": "Visite prescrite de l'extérieur",
     "newpatient": "Type de visite",
-    "newadmittance": "Nouvelle hospitalisation",
+    "newattendance": "Nouvelle hospitalisation",
```

When changing the meaning of a translation, we have to reset the translation for languages other than English

```suggestion
    "newattendance": "New Attendance",
```